workers/scanner/src/main/kotlin/scanner/ScannerRunner.kt

```
@@ -58,6 +58,19 @@ class ScannerRunner(
     ): OrtScannerResult {
         val pluginConfigs = context.resolvePluginConfigSecrets(config.config)
 
+        // If the submodule fetch strategy is set to TOP_LEVEL_ONLY, for git use a plugin config that prevents that
+        // submodules are fetched recursively.
+        val vcsPluginConfigs = if (config.submoduleFetchStrategy == SubmoduleFetchStrategy.TOP_LEVEL_ONLY) {
```

`SubmoduleFetchStrategy.DISBALED` should be handled as well, otherwise the results are confusing when a user sets it. If it's not supported to pass it ORT, it should at least fail.

---

Okay, instead to silently ignore it, throw an exception now.